pkg/resolver/search.go

```diff
@@ -467,9 +571,11 @@ func WhereClauseFilter(input *model.SearchInput) []exp.Expression {
 }
 
 func getKeys(stringArrayMap map[string][]string) []string {
```

Comment:
You could make this more generic and use it from `matchNamespacedResources()`
```suggestion
func getKeys(stringArrayMap map[string]interface{}) []string {
```

Reply:
requires changes in other places where this function is used.